### PR TITLE
translation(zc): add qmix_zh translation

### PR DESCRIPTION
**Pull request by Super1ce**
add qmix_zh

### CODE REVIEW

File: source/12_policies/index_zh.rst [hunk 1]
```diff
@@ -23,6 +23,7 @@
     qrdqn_zh
     rainbow_zh
     iqn_zh
+    qmix_zh
```
Comment (PaParaZz1) [inline source/12_policies/index_zh.rst:26]: 添加多智能体小版块

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/12_policies/qmix_zh.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+2. QMIX 是一种 **无模型、基于价值、离策略、多智能体** 的强化学习方法。
+
+3. QMIX 仅支持 **离散** 动作空间。
+
+4. QMIX 考虑了一种 **部分可观察** 的情景，其中每个智能体只获得个体观察。
+
+5. QMIX 接受 **DRQN** 作为个体价值网络来解决 **部分可观察** 问题。
+
+6. QMIX 使用由 **智能体网络、混合网络、超网络** 组成的架构来表示联合价值函数。 混合网络是一个前馈神经网络，它将智能体网络的输出作为输入并单调地混合它们，产生联合动作值。 混合网络的权重由单独的超网络产生。
```
Comment (PaParaZz1) [inline source/12_policies/qmix_zh.rst:25]: 一些特有的英文名词备注在括号里吧

File: source/12_policies/qmix_zh.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+   * - | 2c_vs_64zg
+       |
+     - 1
+     - .. image:: images/benchmark/QMIX_2c_vs_64zg.png
+     - `config_link_2c_vs_64zg <https://github.com/opendilab/DI-engine/tree/main/dizoo/smac/config/smac_2c64zg_qmix_config.py>`_
+     - | Pymarl(1)
+
+P.S.：
+
+1. 上述结果是通过在五个不同的随机种子 (0, 1, 2) 上运行相同的配置获得的。
```
Comment (PaParaZz1) [inline source/12_policies/qmix_zh.rst:116]: 五个种子，但括号里怎么就三个

Comment (Super1ce) [reply source/12_policies/qmix_zh.rst:116]: 他英文那里就只写了三个。。。

File: source/12_policies/qmix_zh.rst [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+^^^^^^^
+
+概述
+---------
+QMIX 是由 `Rashid et al.(2018) <https://arxiv.org/abs/1803.11485>`_ 提出的，用于在多智能体集中式训练中学习基于全局状态信息的联合动作价值函数，并从集中式端到端框架中提取分布式执行策略。
+QMIX 使用集中式神经网络来估计联合动作值，作为基于局部观察的每个智能体动作值的复杂非线性组合，这为集中式动作价值函数提供了一种新的表示，并保证了集中式和分散式策略之间的一致性。
+
+QMIX 是 `VDN(Sunehag et al. 2017) <https://arxiv.org/abs/1706.05296>`_ 的非线性扩展。
+与 VDN(Value-Decomposition Networks For Cooperative Multi-Agent Learning
+) 相比，QMIX 在训练过程中可以表示更多的额外状态信息，并且可以表示更丰富的动作价值函数类。
```
Comment (Mossforest) [inline source/12_policies/qmix_zh.rst:11]: “表示更多的额外状态信息”有些模糊，应该是指混合网络输入的全局信息让QMIX在训练过程中能够访问并运用更多的额外（智能体观察范围外）信息状态，按照这个意思再改一下；原英文版本也可以同步修改一下

File: source/12_policies/qmix_zh.rst [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+2. QMIX 是一种 **无模型(model-free)、基于价值(value-based)、异策略(off-policy)、多智能体(multi-agent)** 的强化学习方法。
+
+3. QMIX 仅支持 **离散(discrete)** 动作空间。
+
+4. QMIX 考虑了一种 **部分可观察(partially observable)** 的情景，其中每个智能体只获得个体观察。
+
+5. QMIX 接受 **DRQN** 作为个体价值网络来解决 **部分可观察** 问题。
+
+6. QMIX 使用由 **智能体网络(agent networks)、混合网络(a mixing network)、超网络(a hyper-network)** 组成的架构来表示联合价值函数。 混合网络是一个前馈神经网络，它将智能体网络的输出作为输入并单调地混合它们，产生联合动作值。 混合网络的权重由单独的超网络产生。
```
Comment (Mossforest) [inline source/12_policies/qmix_zh.rst:25]: mixing network & hyper network前面的a可以删掉

File: source/12_policies/qmix_zh.rst [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+.. image:: images/marl/qmix.png
+
+QMIX 通过最小化下面的损失函数来训练混合网络：
+
+.. math::
+   y^{tot} = r + \gamma \max_{\textbf{u}^{’}}Q_{tot}(\tau^{'}, \textbf{u}^{'}, s^{'}; \theta^{-})
+
+.. math::
+   \mathcal{L}(\theta) = \sum_{i=1}^{b} [(y_{i}^{tot} - Q_{tot}(\tau, \textbf{u}, s; \theta))^{2}]
+
```
Comment (Mossforest) [inline source/12_policies/qmix_zh.rst:54]: 缺少了一段原文：
"Each weight of the mixing network is produced by a independent hyper-network, which takes the global state as input and outputs the weight of one layer of the mixing network. More details can be found in the original paper [Rashid et al.(2018)](https://arxiv.org/abs/1803.11485)."

**Pull request merged**